mmv1/third_party/terraform/services/logging/resource_logging_bucket_config.go

```diff
@@ -287,6 +314,13 @@ func resourceLoggingBucketConfigRead(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("Error setting cmek_settings: %s", err)
 	}
 
+	indexConfigs, ok := res["indexConfigs"]
+	if ok {
+		if err := d.Set("index_configs", flattenIndexConfigs(indexConfigs.([]interface{}))); err != nil {
```

Comment:
Ah- we want to set this all the time, or else we won't detect if the setting has been removed out of band. Terraform preserves the old value in state if you do nothing in Read.

You can safely remove the `ok`, the flattener will handle a `nil` appropriately.